docs/source/diagnostics-single.rst

```diff
@@ -148,7 +129,14 @@ the number of bytes in the scheduler cache:
 
 
 Example
-^^^^^^^
+-------
+
+.. autosummary::
+   Profiler
+   ResourceProfiler
+   CacheProfiler
+   visualize
```

Comment:
These are already autosummaried individually above, here, and then again at the bottom. Personally I'd only do the one at the bottom, but at a minimum I'd delete this autosummary here.

This also feels like a weird spot for the summary, as its right after the title `Example`, but before the example.

Reply:
I tend to do a little bit of auto-summary in each section with the functions relevant to that section.  I think that this is done particularly well in http://dask.pydata.org/en/latest/futures.html